certbot-apache/certbot_apache/_internal/override_debian.py

```diff
@@ -145,5 +146,27 @@ def _enable_mod_debian(self, mod_name, temp):
             temp, [self.option("dismod"), "-f", mod_name])
         util.run_script([self.option("enmod"), mod_name])
 
+    def restart(self):
+        """Runs a config test and reloads the Apache server.
+
+        :raises .errors.MisconfigurationError: If either the config test
+            or reload fails.
+
+        """
+        self.config_test()
+
+        if not self._ocsp_prefetch:
+            # Try to populate OCSP prefetch structure from pluginstorage
+            self._ocsp_prefetch_fetch_state()
+        if self._ocsp_prefetch:
+            # OCSP prefetching is enabled, so back up the db
+            self._ocsp_prefetch_backup_db()
+
+        self._reload()
```

Comment:
If the reload fails, should we still restore the dbm file?

Reply:
Absolutely, good catch!